appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java

```diff
@@ -1017,9 +958,9 @@ private void setTheme(String theme) {
     }
     if (theme.equals("AppTheme")) {
       blackIcons = false;
-      formWidget.setStylePrimaryName("ode-SimpleMockFormDark");
```

Comment:
Ditto style changes.

Reply:
I'm removing hard-coded styles where I can so that the style can be specified in the template. This one probably broke dark mode functionality, though. I'll revisit.